app/src/main/java/ch/epfl/sdp/musiconnect/Musician.java

```diff
@@ -20,6 +20,7 @@
 
     private static final int MAX_VIDEO_URL_LENGTH = 2048;
 
+    public Musician() {}
```

Comment:
Why the empty constructor? Also, if instantiated with it, videoUrl and instruments will be null rather than empty; that might cause problems